packages/gafl-webapp-service/src/processors/analytics.js

```diff
@@ -7,15 +7,21 @@ export const initialiseAnalyticsSessionData = async (request, previousSessionSta
     clientId = /^.+-(?<clientId>.+)$/.exec(request.query._ga).groups.clientId
   }
   // The user may have an existing session, in which case we need to examine this for attribution and/or clientId
-  await request.cache().helpers.status.set({
-    gaClientId: previousSessionStatusData?.gaClientId || clientId,
-    attribution: previousSessionStatusData?.attribution || {
+  let attribution
+  if (previousSessionStatusData) {
+    attribution = previousSessionStatusData?.attribution
```

Comment:
If you've checked for `previousSessionStatusData`, there's no need to do `previousSessionStatusData?.attribution` as it won't be undefined (although the `attribution` key might be)